playbooks/locator.yml

```
@@ -0,0 +1,19 @@
+---
+# by default this playbook runs in the staging environment
+# to run in production, pass '-e runtime_env=production'
+- name: build the locator site
+  hosts: locator_{{ runtime_env | default('staging') }}
+  remote_user: pulsys
+  become: true
+  vars_files:
+    - ../group_vars/locator/{{ runtime_env | default('staging') }}.yml
+    - ../group_vars/locator/vault.yml
+  roles:
+    - role: locator
+    - role: datadog
```

```suggestion
    - role: datadog
      when: runtime_env | default('staging') == "production"
```
Only install datadog in production. We can add datadog to staging machines temporarily but we should not "build it in" on staging machines.